doc/rtd/reference/datasources/cloudcix.rst

```diff
@@ -0,0 +1,33 @@
+.. _datasource_cloudcix:
+
+CloudCIX
+========
+
+CloudCIX serves metadata through an internal server, accessible at
+``http://169.254.169.254/v1``. The metadata and userdata can be fetched at
+the ``/metadata`` and ``/userdata`` paths respectively.
+
+CloudCIX instances are identified by the dmi product name `CloudCIX`.
+
+Configuration
+-------------
+
+CloudCIX datasource has the following config options:
+
+::
+
+  datasource:
+    CloudCIX:
+      retries: 3
+      timeout: 2
+      wait_retry: 2
```

Comment:
The config values documented should probably match the actual defaults/BUILTIN_CONFIG you set in the DataSourceCIX.

Reply:
Done.